rpc/src/endpoint/broadcast/tx_sync.rs

```diff
@@ -20,11 +18,11 @@ impl Request {
     }
 }
 
-impl rpc::Request for Request {
+impl crate::Request for Request {
     type Response = Response;
 
-    fn method(&self) -> rpc::Method {
-        rpc::Method::BroadcastTxSync
+    fn method(&self) -> crate::Method {
+        crate::Method::BroadcastTxSync
```

Comment:
Couldn't all these be simply `Method::BroadcastTxSync` etc now that they are in a separate crate?

Reply:
We could, but there are some reoccurring names here `Request`, `Response`. Wanna keep it clear where all of them come from.